### DISCUSSION TITLE
quiz 2.2 question 2

### DISCUSSION THREAD

**Discussion by bonosa** [Lightning-AI/dl-fundamentals#16] [Exercises] [answered]
![image](https://user-images.githubusercontent.com/53297623/211865118-a23542a0-fbf6-4d91-a95a-7772f5d55a97.png)

according to the lecture the first column is the feature vector for training set 1...[1  3]

but the correct answer is [1 4] the transpose. Is that correct then? thanks

**Comment by rasbt** [comment, marked as answer]
Thanks for the feedback, and that's an interesting discussion.

The correct answer should actually be "the first training example has feature values 1 and 2". 

It becomes  easier to see if you write/draw it as follows:

<img width="103" alt="Screenshot 2023-01-11 at 10 58 16 AM" src="https://user-images.githubusercontent.com/5618407/211868926-869d3f57-631f-4362-9693-dc90e39ee126.png">


`1 and 3` would be the first feature vector.

**Reply by bonosa** [reply]
now i have to determine how feature values are distinct from feature vectors .

**Reply by rasbt** [reply]
Haha, sorry, I don't want to make it more confusing, but both the row and the column can be called "feature vector". If you consider the row, this is the feature vector of the *first training example*. The vector contains to feature values, the value for feature 1 and the value for feature 2.
The column vector would be the feature 1 values for all training examples.

**Reply by bonosa** [reply]
I think I get it now ! Thanks 🤓
